GoMeet/src/main/webapp/js/manual-time-proposal.js

```diff
@@ -0,0 +1,163 @@
+const MAX_TIME_INPUTS = 5; // Allow up to 5 proposed times
+var timeInputs = 1; // Current number of time inputs on page
+const MEETING_TIME_NAME_PREFIX = 'meeting-time-'; // The prefix of the sessionStorage key name for meeting times
+var timeNameSuffix = 1; // The suffix appended to MEETING_TIME_NAME_PREFIX to make the name unique
+const MAX_TIME_INPUT_REACHED_ALERT = 'Maximum number of proposed times: ' + MAX_TIME_INPUTS.toString();
```

Comment:
Should this be in the error-messages.js file instead?

Reply:
This particular constant depends on MAX_TIME_INPUTS, so both will need to be moved to error-messages.js, but MAX_TIME_INPUTS is not an error message, so I think that's not very good practice. This message is only intended to be used on this JS page, so I think it would be okay to keep it here.

Reply:
Ok makes sense!

Reply:
One possibility is to move all your constants/config into a "constants-config.js" or similar that gets inherited wherever it's needed, but I leave that up to you to do as you see fit.

Reply:
I think we will do some refactoring when everything is merged together to see what's the best way to handle the constants.